src/note.rs

```diff
@@ -19,12 +21,44 @@ pub use self::commitment::{ExtractedNoteCommitment, NoteCommitment};
 pub(crate) mod nullifier;
 pub use self::nullifier::Nullifier;
 
+// We know that `pallas::Base` doesn't allocate internally.
+memuse::impl_no_dynamic_usage!(Rho);
```

Comment:
Nit: position this just below the definition of `Rho`, not above.

Reply:
This is duplicated for how it's done with `Nullifier`.

Reply:
No it isn't:
https://github.com/zcash/orchard/blob/1db974141bdd2dd921a515b1c66df7a7fad91761/src/note/nullifier.rs#L14-L19